tests/backup/backup_upgrade_test.go

```diff
@@ -148,8 +148,16 @@ var _ = Describe("{StorkUpgradeWithBackup}", func() {
 		Step("Upgrade the stork version", func() {
 			log.InfoD("Upgrade the stork version")
 			upgradeStorkImageStr = getEnv(upgradeStorkImage, latestStorkImage)
+			log.Infof("Upgrading stork version on source cluster to %s ", upgradeStorkImageStr)
 			err := upgradeStorkVersion(upgradeStorkImageStr)
-			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of stork version upgrade to - %s", upgradeStorkImageStr))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of stork version upgrade to - %s on source cluster", upgradeStorkImageStr))
+			err = SetDestinationKubeConfig()
+			log.FailOnError(err, "Switching context to destination cluster failed")
+			log.Infof("Upgrading stork version on destination cluster to %s ", upgradeStorkImageStr)
+			err = upgradeStorkVersion(upgradeStorkImageStr)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of stork version upgrade to - %s on destination cluster", upgradeStorkImageStr))
+			err = SetSourceKubeConfig()
```

Comment:
We need to make sure context is switched back to Source cluster in case of failure. So please add it to `JustAfterEach`

Reply:
Done